src/ansys/fluent/core/codegen/tuigen.py

```diff
@@ -295,12 +304,11 @@ def generate(self) -> None:
                         Path(__file__)
                         / ".."
                         / "data"
-                        / f"static_info_{self._version}_{self._mode}.pickle"
```

Comment:
This file is already checked in, we need to regenerate the equivalent .json file.

Reply:
Pickle is unsafe when it deserializes untrusted data. As we have serialized the data here, we can retain the original code.